README.md

```diff
@@ -10,3 +12,27 @@ Edit Ops Added Till Date
 * Digital zoom
 * Alpha blending
 * Unsharpmask
+
+Dependencies
+------------
+* [ArrayFire]
+* [Forge]
```

Comment:
```suggestion
```

Reply:
@9prady9 yes, I also thought the same, but you are linking that library in .pro file.

Reply:
Just to satisfy transitive dependency, which probably isn't needed with newer versions of ArrayFire.